README.md

```diff
@@ -39,23 +42,22 @@ jobs:
       - uses: fastify/github-action-merge-dependabot@v1
         if: ${{ github.actor == 'dependabot[bot]' && github.event_name == 'pull_request' }}
         with:
-          github-token: ${{secrets.github_token}}
+          github-token: ${{secrets.GITHUB_TOKEN}}
```

Comment:
why this change?

Reply:
because it was wrong https://docs.github.com/en/actions/reference/authentication-in-a-workflow